wazo_calld/plugins/applications/stasis.py

```diff
@@ -88,16 +88,18 @@ def initialize(self, token):
         logger.debug('Stasis applications initialized')
 
     def stasis_start(self, event_objects, event):
-        args = event.get('args', [])
         application_uuid = AppNameHelper.to_uuid(event.get('application'))
-        if not application_uuid or len(args) < 1:
+        if not application_uuid:
             return
 
-        command = args[0]
+        if not event['args']:
```

Comment:
we cannot control the argument for a user outgoing call? It will be more explicit
Or in other words, why the logic from the first commit has been removed?

Reply:
The outgoing call handler is not called by the dialplan, it replaces the context on the endpoint. This does not allow any arguments to be passed down to the stasis application.

I refactored the handler hoping that it was clear that this stasis_start handler was different from others.